webpack.config.js

```diff
@@ -75,10 +77,60 @@ const externals = {
 	};
 } );
 
+/**
+ * Webpack plugin for handling specific template tags in Webpack configuration
+ * values like those supported in the base Webpack functionality (e.g. `name`).
+ *
+ * @see webpack.TemplatedPathPlugin
+ */
+class CustomTemplatedPathPlugin {
```

Comment:
Do we plan to extract it and move to `WordPress/packages` at some point? It might be useful for other repositories.

Reply:
>Do we plan to extract it and move to `WordPress/packages` at some point? It might be useful for other repositories.

Yeah, seems like a generally useful plugin.